packages/edit-site/src/components/use-template-title/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { store as editorStore } from '@wordpress/editor';
+
+function useTemplateTitle( templateType, templateId ) {
```

Comment:
Is this hook useful given we now only have one component using it?

Reply:
Yeah, probably not 😅. I can remove it.